lib/pharos/scripts/configure-etcd.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 etcd_healthy() {
-  response=$(curl -s --cacert /etc/pharos/pki/ca.pem --cert /etc/pharos/pki/etcd/client.pem --key /etc/pharos/pki/etcd/client-key.pem https://${PEER_IP}:2379/health)
+  response=$(curl -s --noproxy "*" --cacert /etc/pharos/pki/ca.pem --cert /etc/pharos/pki/etcd/client.pem --key /etc/pharos/pki/etcd/client-key.pem https://${PEER_IP}:2379/health)
```

Comment:
This could be something like `--no-proxy "${PEER_IP}"`. Theoretically I suppose the `https://${PEER_IP}:2379/health` could redirect somewhere. Maybe far fetched.